### PR TITLE
Windows XP: backward compatibility fix

### PR DESCRIPTION
**Pull request by gpotter2**
fixes https://github.com/secdev/scapy/issues/1309

Really hard to test :/ Changed lines are in a piece of VBS metric

PS: I have setup an Windows XP Vbox to debug this, so I could test it

### CODE REVIEW

**Comment by gpotter2** [comment]
@guedou Hi ! Fix confirmed by the issuer. Should be ready to merge :)

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,5 @@ MANIFEST
 *.egg-info/
 scapy/VERSION
 test/*.html
+.tox
```
Comment (p-l-) [inline .gitignore]: Why did you add that?

Comment (gpotter2) [reply .gitignore]: @guedou told me to debug tox using `tox -e flake8` which generates this folder, detected by github

Comment (p-l-) [reply .gitignore]: OK.
